Three/include/CGAL/Three/Scene_interface.h

```diff
@@ -45,13 +45,13 @@ class Scene_group_item;
  */
 enum RenderingMode { Points = 0,
                      PointsPlusNormals,
+                     ShadedPoints,
                      Splatting,
                      Wireframe, 
                      Flat,
                      FlatPlusEdges,
                      Gouraud,
-                     LastRenderingMode = Gouraud,
-                     NumberOfRenderingMode = LastRenderingMode+1 };
+                     NumberOfRenderingMode};
```

Comment:
I look at the code. `NumberOfRenderingMode` is used by `Scene.cpp` and `Scene_item.cpp`. Those two must be recompiled (they are part of the `demo_framework` library). The headers or code of other scene items does not depend on the value of `NumberOfRenderingMode` as far as I have seen.